docs/extend/modular-resources/configure.md

```diff
@@ -193,46 +194,89 @@ The custom model is configured as a component with the name "my-realsense".
 {{% /tab %}}
 {{% /tabs %}}
 
-## Add a local module to your robot
+## Local modules
 
-If you are developing your own modular resource, and intend to deploy it to your robot locally, first follow [these steps](/extend/modular-resources/create/) to code your own module and generate an executable.
-If you are using a pre-built modular resource, make sure you install the module and determine the filename of [the module's executable](/extend/modular-resources/create/#compile-the-module-into-an-executable).
+If you wish to add a module to your robot without uploading it to the Viam registry, you can add your module as a *local module*.
 
-Follow these steps to configure a module and its modular resources locally:
+You can add your own custom modules as local modules, or you can add pre-built modules written by other Viam users.
 
-1. [Save the executable](#make-sure-viam-server-can-access-your-executable) in a location your `viam-server` instance can access.
-2. [Add a **module**](#configure-your-module) referencing this executable to the configuration of your robot.
-3. [Add a new component or service](#configure-your-modular-resource) referencing the modular resource provided by the configured **module** to the configuration of your robot.
+### Prepare a local module
 
-### Make sure `viam-server` can access your executable
+First determine the module you wish to add as a local module:
 
-Ensure that your module executable is saved where the instance of `viam-server` behind your robot can read and execute it.
+- If you are adding your own custom module, be sure that you have followed the steps to [create your own module](/extend/modular-resources/create/) to code and compile your module and generate an executable.
+- If you are using a pre-built module, make sure you have installed the module and determined the filename of [the module's executable](/extend/modular-resources/create/#compile-the-module-into-an-executable).
 
-For example, if you are running `viam-server` on an Raspberry Pi, you'll need to save the module on the Pi's filesystem.
+Then, ensure that `viam-server` is able to find and run the executable:
 
-Obtain the real (absolute) path to the executable file on your computer's filesystem by running the following command in your terminal:
+- Ensure that the module executable is saved to a location on the filesystem of your robot that `viam-server` can access.
+  For example, if you are running `viam-server` on an Raspberry Pi, you must save the module executable on the Pi's filesystem.
+- Ensure that this file is executable (runnable) with the following command:
```

Comment:
```suggestion
- Ensure that this file is executable with the following command:
```
(opt) I don't know if we need this clarification, but up to you really!!

Reply:
Gotcha, gotcha, definitely fair. Only because all other uses of `executable` surrounding it refer to the file itself, this is the only one that refers to a feature of that file: whether it is "runnable" (executable) or not. It's also why I call it `file` here instead of ... `ensure the executable is executable` 😅 

I don't like `runnable` per se -- you right! -- but I think we might disambiguate for a few users with this explicitness. 🤷